src/Illuminate/Support/Str.php

```diff
@@ -112,6 +112,24 @@ public static function contains($haystack, $needles)
         return false;
     }
 
+    /**
+     * Determine if a given string contains all array values.
+     *
+     * @param  string  $haystack
+     * @param  string|array  $needles
+     * @return bool
+     */
+    public static function containsAll($haystack, $needles)
```

Comment:
I'd would think this could explicitly require an array. Otherwise you would just use `Arr::contains()` for a single item.

Reply:
Thanks for the feedback, Tim. I have asked myself the same question on whether a string should or should not be allowed. I thought it wouldn't "hurt" to keep the functionality and possibly could come in handy at times.

But you're right; logically, `Str::contains()` would be a more suitable call for that.

Definitely open for a discussion here on whether to keep or to remove it!